bubblegum/program/README.md

```diff
@@ -1,4 +1,438 @@
-# Bubblegum - NFTs in Merkle Trees
-Supports decompressing bubblegum NFTs into either Tokenkeg tokens or Token22 tokens. The benefit of decompressing a bubblegum 
-NFT is that normal tokens can be moved into a custodial wallet and freely transferred without relying on RPC nodes to serve 
-your NFT data from an off-chain database.
+# Bubblegum - Compressed Metaplex NFTs
+
+> ⚠️ **Bubblegum is currently experimental and has not been formally audited. Use in production
+> at your own risk.**
+
+## Overview
+
+`Bubblegum` is the Metaplex Protocol program for creating and interacting with compressed Metaplex NFTs.  Compressed NFTs are secured on-chain using Merkle trees.
+
+With Bubblegum you can:
+* Create a tree
+* Delegate authority for a tree.
+* Mint a compressed NFT to a tree.
+* Verify/unverify creators
+* Verify/unverify membership of an NFT to a Metaplex Verified Collection.
+* Transfer ownership of an NFT.
+* Delegate authority for an NFT.
+* Burn an NFT.
+* Redeem an NFT and decompress it into an uncompressed Metaplex NFT.
+
+## Background
+
+Compressed NFTs differ from uncompressed NFTs in where their state (metadata, owner, etc.) is stored.  For uncompressed NFTs, all state data is stored in on-chain accounts.  This tends to be expensive at scale.  Compressed NFTs save space by encoding the state data into a Merkle tree.  This means that the detailed account data is not stored on-chain, but in data stores managed by RPC providers.
+
+Compressed NFTs are secured on-chain by the hashing of the state data when it is added to the Merkle tree.  The Merkle root is a hash that cryptographically secures the state data for all of the leaves (NFTs) contained in the tree.
+
+In the unlikely scenario that all RPC providers were to lose their data stores, the off-chain state of compressed NFTs could be recovered by replaying transactions (provided that the given tree was started from the beginning).
+
+Compressed NFTs can also be losslessly decompressed into uncompressed Metaplex NFTs.  Decompression will cost rent for the token-metadata Metadata and Master Edition accounts that need to be created.
```

Comment:
> Decompression will cost rent for the token-metadata Metadata and Master Edition accounts that need to be created.

Nit: This reads oddly to me. Maybe:

> Decompression will cost rent for the `token-metadata` program Metadata and Master Edition accounts that need to be created.

Or:

> Decompression will cost rent for the token-metadata program accounts that need to be created. 

Or:

> Decompression will cost rent for the Metadata and Master Edition `token-metadata` program accounts that need to be created.

Reply:
done